admin/spec/service/check-complete.service.spec.js

```diff
@@ -38,6 +40,7 @@ describe('check-complete.service', () => {
       spyOn(markingService, 'mark').and.returnValue(Promise.resolve())
       spyOn(jwtService, 'decode').and.returnValue({ sub: 1 })
       spyOn(completedCheckDataService, 'sqlFindOneByCheckCode')
+      spyOn(checkDataService, 'sqlFindOneByCheckCode').and.returnValue(Promise.resolve(checkMock))
```

Comment:
@jon-shipley what if the StartedAt date is not valid? don't we need a test for that as well

Reply:
Yes.  I didn't bother as the pupil status re-work may render this irrelevant

Reply:
Well as long as it is in future debt it's fine.